src/views/system-security-plans/__tests__/SystemSecurityPlanLeverageView.spec.ts

```
@@ -0,0 +1,254 @@
+    expect(wrapper.text()).toContain('GovCloud Baseline');
+    expect(wrapper.text()).toContain('v2');
+    expect(wrapper.text()).toContain('1 item: ac-1');
+  });
+
+  it('never fetches anything upstream-scoped — only the flat catalog', () => {
+    offeringsData.current = [makeOffering()];
+    mountView();
+    expect(fetchedUrls).toEqual(['/api/oscal/ssp-export-offerings']);
+    for (const url of fetchedUrls) {
+      expect(url).not.toContain('ssp-upstream-1');
+      expect(url).not.toMatch(/system-security-plans\/(?!undefined)/);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)